src/net/http.js

```diff
@@ -45,6 +45,8 @@ Object.assign(pc, function () {
         '.dds'
     ];
 
+    Http.retryDelay = 100;
```

Comment:
Won't this be shared for all instances of running engines?

Reply:
Yeah it's not really document or meant to be public it's mostly to make this unit testable.